### PR TITLE
CONSOLE-3927: add console.resource/details-item-s to details page

### PR DESCRIPTION
**Pull request by rhamilto**
The items in the right column are added via `console.resource/details-item`.

localhost_9000_k8s_ns_robb_stable example com~v1~CronTab_my-new-cron-object_yaml

Empty state

localhost_9000_k8s_ns_robb_stable example com~v1~CronTab_my-new-cron-object_

### CODE REVIEW

**Review by TheRealJon — APPROVED** [review]
/lgtm

**Comment by rhamilto** [comment]
QE Approver:
/assign @yapei
Docs Approver:
/assign @opayne1
PX Approver:
/assign @RickJWagner

**Comment by rhamilto** [comment]
@jhadvig, @TheRealJon, CI is not building an image with these changes when running Cypress.  Any idea how to resolve?  I think this has been an issue since the very beginning of this repo but it went unnoticed since the tests worked with the default list and details pages.

**Comment by rhamilto** [comment]
> @jhadvig, @TheRealJon, CI is not building an image with these changes when running Cypress. Any idea how to resolve? I think this has been an issue since the very beginning of this repo but it went unnoticed since the tests worked with the default list and details pages.

Hmm.  https://github.com/openshift/release/blob/4add02f5ef26c725087f17b7cc95c9c3892914d3/core-services/image-mirroring/supplemental-ci-images/mapping_supplemental_ci_images_ci#L103 is the culprit?

File: package.json
Comment (rhamilto) [inline package.json]: Had to drop this to get the build to run with PF5.  @vojtechszocs

**Review by jhadvig — APPROVED** [review]
/lgtm

**Comment by rhamilto** [comment]
I removed the Cypress test updates so we can move forward with the changes and debug the test issues in a follow on.

**Comment by rhamilto** [comment]
Blocked by https://issues.redhat.com/browse/DPTP-3894

**Comment by yapei** [comment]
tested the new changes and working as expected: `CronSpec` `Image` `Replicas` item on CronTab details page shows correct value and `-` when they the values are empty
Screenshot 2024-02-20 at 5 29 21 PM
/label qe-approved

**Review by TheRealJon — APPROVED** [review]
/lgtm

**Pull request merged**
